.tools/weathertop/backend/src/main/java/com/weathertop/lambda/HandleEventBridgeInspector.java

```diff
@@ -0,0 +1,379 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.weathertop.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.weathertop.service.EcsEventBridgeInspector;
+import java.util.Map;
+
+public class HandleEventBridgeInspector implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+
+    private APIGatewayProxyResponseEvent createErrorResponse(int statusCode, String message) {
+        return new APIGatewayProxyResponseEvent()
+                .withStatusCode(statusCode)
+                .withHeaders(Map.of(
+                        "Access-Control-Allow-Origin", "*",
+                        "Access-Control-Allow-Methods", "GET,OPTIONS",
+                        "Access-Control-Allow-Headers", "Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token"
+                ))
+                .withBody("{\"error\": \"" + message.replace("\"", "\\\"") + "\"}");
+    }
+
+    @Override
+    public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent event, Context context) {
+        LambdaLogger logger = context.getLogger();
+
+        try {
+            Map<String, String> queryParams = event.getQueryStringParameters();
+            if (queryParams == null || !queryParams.containsKey("language")) {
+                return createErrorResponse(400, "Missing required query parameter: language");
+            }
+
+            String language = queryParams.get("language").toLowerCase();
+            logger.log("Received language: " + language);
+
+            switch (language) {
+                case "java": {
+                    logger.log("Processing Java SDK EventBridge Inspector");
+
+                    EcsEventBridgeInspector inspector = new EcsEventBridgeInspector(
+                            "MyJavaWeathertopCluster",
+                            "WeathertopJava",
+                            "WeathertopJavaLogs",
+                            "ecs-java-schedule"
+                    );
+
+                    Map<String, Object> data = inspector.inspect();
```

Comment:
These 12 lines of code are repeated 10 times, can they be consolidated by using variables within the switch and then the inspector/json/response return all in one place at the end to make it more maintainable?